yt/visualization/volume_rendering/scene.py

```diff
@@ -226,8 +226,25 @@ def render(self, camera=None):
         self._last_render = bmp
         return bmp
 
+    def _check_render(self, fname, render=True):    
```

Comment:
how about `_check_then_render_missing` ? or `_render_if_missing`?

Reply:
went with `_render_if_missing` :)

Reply:
> If your function's name has conjunctions, maybe it's time to refactor

Can't remember where I heard that exactly but that's why I can't find a good name for this !
In particular `_render_if_missing` feels wrong since it can also render if passed `render=True`...

I'm thinking the logic could be slightly changed so that the function would be named `_sanitize_render_switch` and called early in `save()` and `save_annotated()`, leaving the actual rendering part to those functions (which would also make them clearer, even if this implies repeating 2 lines). What do you think ?